src/components/views/rooms/RoomSettings.js

```diff
@@ -942,6 +947,10 @@ module.exports = React.createClass({
             </AccessibleButton>;
         }
 
+        const devtoolsButton = <AccessibleButton className="mx_RoomSettings_devtoolsButton" onClick={this._openDevtools}>
+            { _t("Open Devtools") }
+        </AccessibleButton>;
```

Comment:
Can we get it behind a boolean settings flag so non-powerusers aren't confused?

Reply:
It is stuck right under advanced? I don't know if a settings flag is going to do much to stop confusion.

Reply:
For reference (for you, or others, or no one):

* Add a setting to `Settings.js` - autoplay GIFs is a good one to copy for this ( https://github.com/matrix-org/matrix-react-sdk/blob/167746f41d891d053643e061a583489f2826755a/src/settings/Settings.js#L138-L142 )
* Add the setting to `UserSettings.js` - https://github.com/matrix-org/matrix-react-sdk/blob/167746f41d891d053643e061a583489f2826755a/src/components/structures/UserSettings.js#L63-L85
* Use the setting in an if statement like `if (SettingsStore.getValue("yourSettingName"))`

Reply:
gah, github only now showed me your comment, sorry.

Even though it is under advanced, people will probably still click it and become confused.

Reply:
Yes, I wrote the autoplay GIFs feature so I know how settings works :p